docker/Dockerfile

```diff
@@ -45,14 +46,15 @@ RUN mkdir -p /opt/mesosphere/ && \
     unzip bootstrap.zip && \
     rm bootstrap.zip
 
+# JVM/JCE
 RUN mkdir -p /usr/lib/jvm/ && \
     cd /usr/lib/jvm && \
-    curl -L -O https://downloads.mesosphere.com/java/jre-8u152-linux-x64-jce-unlimited.tar.gz && \
-    tar zxf jre-8u152-linux-x64-jce-unlimited.tar.gz && \
-    rm jre-8u152-linux-x64-jce-unlimited.tar.gz
+    curl -L -O https://downloads.mesosphere.com/java/jre-8u131-linux-x64-jce-unlimited.tar.gz && \
+    tar zxf jre-8u131-linux-x64-jce-unlimited.tar.gz && \
+    rm jre-8u131-linux-x64-jce-unlimited.tar.gz
 
 ENV BOOTSTRAP /opt/mesosphere/bootstrap
-ENV JAVA_HOME /usr/lib/jvm/jre1.8.0_152
+ENV JAVA_HOME /usr/lib/jvm/jre1.8.0_131
```

Comment:
should we update to 151?

Reply:
151 was broken! 131 was the answer, also this is what we're using (currently at least) in the SDK services.

Reply:
ok good to know!